.github/workflows/store_latest_datasets_cronjob.yml

```diff
@@ -1,16 +1,20 @@
 name: Store latest datasets cronjob
 
 on:
+  workflow_dispatch:
   schedule:
     - cron: "0 0 * * *"
 
+env:
+  URL_STORAGE_PREFIX: "https://storage.googleapis.com/storage/v1/b/mdb-latest/o"
```

Comment:
It's ok for now, but we should consider another way to specify this prefix in case we ever change the location of the datasets.